account_payment_order/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
```

Comment:
does not lack the import of '_' for translations?

Reply:
_ is only used when translating strings from python code/methods
strings defined on fields in python or defined in XML are automatically translatable
seems like this file does not contain python methods, only field definitions
so _ for translations is not needed